tests/client_test.go

```diff
@@ -14,17 +14,56 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package trello
+package tests
 
 import (
+	"os"
 	"testing"
 
 	"github.com/VojtechVitek/go-trello"
+
+	. "github.com/franela/goblin"
```

Comment:
Why not github.com/onsi/ginkgo, is it very different?

tbh, I'm not familiar with these pkgs -- I mostly use https://github.com/stretchr/testify